### PR TITLE
[ASCellNode] "selectedBackgroundView" property

### PR DESCRIPTION
**Pull request by acedened**
Just a simple little propetry

### CODE REVIEW

File: AsyncDisplayKit/ASTableView.mm [hunk 1]
```diff
@@ -806,6 +806,7 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
     cell.node = node;
     cell.backgroundColor = node.backgroundColor;
     cell.selectionStyle = node.selectionStyle;
+    cell.selectedBackgroundView = node.selectedBackgroundView;
```
Comment (Adlai-Holler) [inline AsyncDisplayKit/ASTableView.mm]: Would you be willing to add the same code into `-[ASCollectionView collectionView:cellForItemAtIndexPath:]`?

**Comment by Adlai-Holler** [comment]
Jenkins, test this please

File: AsyncDisplayKit/ASTableView.mm (hunk 1 above)
Comment (acedened) [inline AsyncDisplayKit/ASTableView.mm]: Oh, yes. Forgot about that.

**Comment by acedened** [comment]
@Adlai-Holler Done.

**Comment by Adlai-Holler** [comment]
Jenkins add to whitelist and test this please.

**Comment by acedened** [comment]
Fixed!

**Pull request merged**